File: PilotProject/PilotProject/Models/Flowsheet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace PilotProject.Models
{
    public class Flowsheet
    {
        public int flowsheetID { get; set; }
        public string customField { get; set; }
        public int durationHours { get; set; }
        public int startHour { get; set; }
        public DateTime date { get; set; }

        public int medicID { get; set; }
        public virtual Medic doctor { get; set; }

        public int hospitalizationID { get; set; }
        public virtual Hospitalization hospitalization { get; set; }

        public virtual ICollection<ParameterBase> parameters { get; set; }
    }
}

File: PilotProject/PilotProject/Scripts/app/flowsheet.js
//Flowsheet model

function Flowsheet(data) {
    this.id = data.flowsheetID;
    this.customField = data.customField;
    this.durationHours = data.durationHours;
    this.startHour = data.startHour;
    this.date = data.date;
    this.medicID = data.medicID;
    this.doctor = data.doctor;
}

function Medic(data) {
    console.log(data);
    this.id = data.medicID;
    this.nameFull = data.nameFull;
}

function MedicListViewModel() {
    var self = this;
    self.medics = ko.observableArray([]);
    $.getJSON("/api/medicsweb", function (allData) {
        var mappedTasks = $.map(allData.$values, function (item) {
            return new Medic(item)
        });
        self.medics(mappedTasks);
    });
}

ko.applyBindings(new MedicListViewModel());
